custom_components/ha_creality_ws/camera.py

```
@@ -773,13 +777,27 @@ async def async_handle_async_webrtc_offer(
-                "ha_creality_ws: go2rtc client error handling WebRTC offer: %s",
-                exc, exc_info=True
+                "ha_creality_ws: go2rtc client error handling WebRTC offer for stream '%s': %s",
+                self._stream_name, go2rtc_err, exc_info=True
             )
+
+            # Attempt recovery by invalidating the stream to force reconfiguration next time
+            if self._stream_name:
+                _LOGGER.warning("ha_creality_ws: Invalidating stream '%s' due to go2rtc error", self._stream_name)
+                try:
+                    await self._go2rtc_client.streams.delete(self._stream_name)
+                except Exception as cleanup_exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)